tests/software_tests/packet/test_abstract_packet.py

```
@@ -52,20 +57,22 @@ def teardown_method(self):
 
     # __init__
 
-    @pytest.mark.parametrize("frame, direction, transmission_time, transmission_timestamp", [
-        (Mock(), Mock(), Mock(), Mock()),
-        ("Some frame", "Some direction", "Some time", "Some timestamp"),
+    @pytest.mark.parametrize("frame, direction, transmission_time, transmission_timestamp, transmission_native_timestamp", [
```

Ruff (0.16.2)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)